.babelrc

```diff
@@ -1,5 +1,8 @@
 {
-    "presets":[
-        "latest"
-    ]
+  "presets": ["react", "es2015"],
+  "env": {
+    "development": {
+      "presets": ["react-hmre"]
+    }
+  }
 }
```

Comment:
add an end of line here. Also delete the `.DS_Store` above and add it to your `.gitignore` file.